building-srpm-in-copr/README.md

```diff
@@ -90,3 +85,27 @@ copr_client.build_proxy.create_from_custom(ownername="lbarczio", projectname="og
                            script_builddeps=["git", "packit", "python3-wheel", "python3-pip", "python3-setuptools",
                             "python3-setuptools_scm", "python3-setuptools_scm_git_archive"])
 ```
+
+### Plan how to build SRPMs in Copr in Packit Service
+
+- create CLI command or only Packit API method for preparing sources (would mostly use existing functionality, described above)
```

Comment:
such command would be also really helpful when testing out things and especially when configuring the archive generation